### PR TITLE
Add AKS specs for apps

### PR DESCRIPTION
**Pull request by stgleb**

### CODE REVIEW

**Comment by stgleb** [comment]
Build passed https://jenkins.portworx.co/job/PX-Backup/job/gleb-aks-backup-job/148

**Comment by stgleb** [comment]
Job passed: https://jenkins.portworx.co/job/PX-Backup/job/gleb-aks-backup-job/173/

File: drivers/scheduler/k8s/k8s.go [hunk 1]
```diff
@@ -56,6 +56,8 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+
```
Comment (vinayakshnd) [inline drivers/scheduler/k8s/k8s.go:59]: remove extra line

File: drivers/scheduler/k8s/k8s.go [hunk 2]
```diff
@@ -74,6 +76,13 @@ const (
 	ZoneK8SNodeLabel = "failure-domain.beta.kubernetes.io/zone"
 	// RegionK8SNodeLabel is node label describing region of the k8s node
 	RegionK8SNodeLabel = "failure-domain.beta.kubernetes.io/region"
+
```
Comment (vinayakshnd) [inline drivers/scheduler/k8s/k8s.go:79]: same here

File: deployments/deploy-ssh.sh [hunk 3]
```diff
@@ -369,6 +376,8 @@ spec:
       value: "${AZURE_TENANTID}"
     - name: PROVIDER
```
Comment (vinayakshnd) [inline deployments/deploy-ssh.sh:377]: Can we change this to VOLUME_PROVIDER to be more specific?

**Pull request merged**